pages/DetailPage.js

```diff
@@ -0,0 +1,45 @@
+import { campingTrips } from "../data/campingTrips";
+import { Container, Card, Header, Footer } from "../styles/styles";
+import Image from "next/image";
+import Link from "next/link";
+
+export default function DetailPage() {
+  const trip = campingTrips[0];
```

Comment:
I personally would call the singular and the plural of the data the same: So I would do campingTrips and campingTrip or trips and trip. For me this would be semantically clearer, but of course it's not necessary :)